content/en/platform/corda/5.0/deploying-operating/config/secrets.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Configuration Secrets"
+date: '2023-05-12'
+menu:
+  corda5:
+    identifier: corda5-cluster-secrets
+    parent: corda5-cluster-config
+    weight: 3000
+section_menu: corda5
+---
+
+The Corda configuration system allows for any string configuration value to be marked as “secret”. When this configuration value is used, Corda delegates the resolution of this value to one of the following configured secrets lookup service:
+* [Default Secrets Service]({{< relref "#default-secrets-service">}})
+* [External Secrets Service]({{< relref "#external-secrets-service--enterprise-icon">}}) {{< enterprise-icon >}}
+
+{{< note >}}
+Any configuration items can be configured as sensitive or not. It is up to you to decide if a particular configuration item should be treated as sensitive.
+{{< /note >}}
+
+## Default Secrets Service
+
+Corda 5 provides a default secrets lookup service. Implementation of this service is in the form of a service that uses symmetric encryption so that the value can be stored encrypted at rest and decrypted with a key derived from a configured salt and passphrase when needed. The salt and passphrase must be specified in the [deployment configuration]({{< relref "../deploying/deploying.md#default-secrets-service" >}}).
+
+For example, the following is a standard configuration:
+
+```
+{
+  "user": "name",
+  "pass": "mypassword"
+}
+```
+
+You can specify the `pass` value as a secret using the `configSecret` value, as follows:
+
+```
+{
+  "user": "name",
+  "pass": {
+    "configSecret": {
+      "encryptedSecret": "<encrypted-password>"
+    } 
+  }
+}
+```
+
+You can use the Corda CLI <a href = "../../reference/corda-cli/secret-config.md">`secret-config` command</a> to generate the configuration for an encrypted value.
```

Comment:
relref usage here ?

Reply:
The code font styling doesn't work with markdown links.